jest.config.js

```diff
@@ -15,37 +21,43 @@ module.exports = {
       statements: 80,
     },
   },
-  transform: {
-    "\\.js$": "babel-jest",
-    "\\.ts$": "ts-jest"
-  },
   collectCoverageFrom: [
-    '**/*/src/**/*.ts',
-    '!**/index.ts',
-    '!**/__integrationtests__/**',
-    '!**/__mocks__/**',
-    '!**/__tests__/**',
-    '!**/lib/**',
-    '!**/test/**',
-    '!**/kilt/*',
-    '!**/types/**/*',
-    '!**/SDKErrors.ts',
-    '!utils/src/json-schema/*',
-    '!testing/**',
-    '!augment-api/**',
-    '!type-definitions/**',
-    '!**/*.chain.ts',
-    '!did/src/Did.chain.ts',
-    '!did/src/Did.rpc.ts',
-    '!did/src/Did.utils.ts',
-    '!utils/src/jsonabc.ts',
-    '!core/src/utils.ts',
+    'packages/*/src/**',
+  ],
+  coveragePathIgnorePatterns: [
+    '/node_modules/',
+    '/lib/',
+    '/tests/',
+    'packages/types/',
+    'packages/augment-api/',
+    'packages/type-definitions/',
+    'packages/utils/src/json-schema/',
+    'packages/core/src/kilt/',
+    'index.ts',
+    'types.ts',
+    '.chain.ts',
+    'SDKErrors.ts',
+    'Did.chain.ts',
+    'Did.rpc.ts',
+    'Did.utils.ts',
+    'jsonabc.ts',
+    'packages/core/src/utils.ts',
   ],
-  resolver: "ts-jest-resolver",
-  rootDir: 'packages',
-  coverageDirectory: 'coverage',
-  moduleDirectories: [
-    "node_modules",
-    "packages/*/src"
-  ]
 }
+
+module.exports = {
+  ...common,
```

Comment:
Is common needed here?

Reply:
it is currently hard to see which setting is expected in which context, I had a lot of issues with jest ignoring some settings. This avoids them by defining all options both within the projects context and the outer context.

Reply:
Weird how Jest does that :/